src/ripple/core/JobTypes.h

```diff
@@ -67,46 +67,55 @@ class JobTypes
-        add(jtPUBLEDGER,     "publishNewLedger",       maxLimit,  3000ms,  4500ms);
-        add(jtTXN_DATA,      "fetchTxnData",                  5,     0ms,     0ms);
-        add(jtWAL,           "writeAhead",             maxLimit,  1000ms,  2500ms);
-        add(jtVALIDATION_t,  "trustedValidation",      maxLimit,   500ms,  1500ms);
-        add(jtWRITE,         "writeObjects",           maxLimit,  1750ms,  2500ms);
-        add(jtACCEPT,        "acceptLedger",           maxLimit,     0ms,     0ms);
-        add(jtPROPOSAL_t,    "trustedProposal",        maxLimit,   100ms,   500ms);
-        add(jtSWEEP,         "sweep",                         1,     0ms,     0ms);
-        add(jtNETOP_CLUSTER, "clusterReport",                 1,  9999ms,  9999ms);
-        add(jtNETOP_TIMER,   "heartbeat",                     1,   999ms,   999ms);
-        add(jtADMIN,         "administration",         maxLimit,     0ms,     0ms);
-        add(jtMISSING_TXN,   "handleHaveTransactions",     1200,     0ms,     0ms);
-        add(jtREQUESTED_TXN, "doTransactions",             1200,     0ms,     0ms);
-
-        add(jtPEER,          "peerCommand",                   0,   200ms,  2500ms);
-        add(jtDISK,          "diskAccess",                    0,   500ms,  1000ms);
-        add(jtTXN_PROC,      "processTransaction",            0,     0ms,     0ms);
-        add(jtOB_SETUP,      "orderBookSetup",                0,     0ms,     0ms);
-        add(jtPATH_FIND,     "pathFind",                      0,     0ms,     0ms);
-        add(jtHO_READ,       "nodeRead",                      0,     0ms,     0ms);
-        add(jtHO_WRITE,      "nodeWrite",                     0,     0ms,     0ms);
-        add(jtGENERIC,       "generic",                       0,     0ms,     0ms);
-        add(jtNS_SYNC_READ,  "SyncReadNode",                  0,     0ms,     0ms);
-        add(jtNS_ASYNC_READ, "AsyncReadNode",                 0,     0ms,     0ms);
-        add(jtNS_WRITE,      "WriteNode",                     0,     0ms,     0ms);
+        //                                                           avg     peak
+        //  JobType               name                    limit    latency  latency
+        add(jtPACK,              "makeFetchPack",               1,     0ms,     0ms);
+        add(jtPUBOLDLEDGER,      "publishAcqLedger",            2, 10000ms, 15000ms);
+        add(jtVALIDATION_ut,     "untrustedValidation",  maxLimit,  2000ms,  5000ms);
+        add(jtTRANSACTION_l,     "localTransaction",     maxLimit,   100ms,   500ms);
+        add(jtREPLAY_REQ,        "ledgerReplayRequest",        10,   250ms,  1000ms);
+        add(jtLEDGER_REQ,        "ledgerRequest",               5,     0ms,     0ms);
+        add(jtPROPOSAL_ut,       "untrustedProposal",    maxLimit,   500ms,  1250ms);
+        add(jtREPLAY_TASK,       "ledgerReplayTask",     maxLimit,     0ms,     0ms);
+        add(jtLEDGER_DATA,       "ledgerData",                  5,     0ms,     0ms);
```

Comment:
The limit for this job type used to be 4. Now it's 5. Was the change intentional?